domain/consensus/processes/blockprocessor/validateandinsertblock.go

```diff
@@ -221,7 +265,7 @@ func (bp *blockProcessor) hasValidatedOnlyHeader(blockHash *externalapi.DomainHa
 		return false, err
 	}
 
-	return status == externalapi.StatusHeaderOnly, nil
+	return status != externalapi.StatusInvalid, nil
```

Comment:
Rename function to hasValidatedHeader.
Also, to avoid ambiguity, add comment on function that says that it returns true if both header and body were validated